packages/ui/components/footer/footer.tsx

```diff
@@ -105,7 +105,7 @@ export function FooterContent(): JSX.Element {
               title="janus-idp.io homepage"
               href="/"
             >
-              <HeaderLogo />
+              <JanusLogo />
```

Comment:
HeaderLogo looks fine to me. If one day Janus is renamed, we don't have to rename these files.

Reply:
`JanusLogo` is more accurate to what the component is. It could also be used elsewhere that is not in the header. 

VS Code renames all instances of a variable name when you press F2, so that shouldn't be an issue in the future.